Sources/InjiVcRenderer/PreProcessor.swift

```diff
@@ -0,0 +1,172 @@
+import Foundation
+import pixelpass
+
+public class PreProcessor {
+    
+    public init() {}
+    
+    private let QRCODE_IMAGE_TYPE = "data:image/png;base64,"
+
+    public func preProcessVcJson(vcJsonString: String, svgTemplate: String) -> [String: Any] {
+        var vcJsonObject = try! JSONSerialization.jsonObject(with: Data(vcJsonString.utf8), options: []) as! [String: Any]
```

Comment:
lets avoid forced unwrap.
We can refer this.
https://cocoacasts.com/what-is-the-difference-between-try-try-and-try

Reply:
Updated